ci/tasks/scan-and-submit-sboms.yml

```diff
@@ -0,0 +1,63 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: ((concourse-team/dev_image_registry.url))/((platform-automation/main/docker.ci-repository))
+    username: ((concourse-team/dev_image_registry.username))
+    password: ((concourse-team/dev_image_registry.password))
+    tag: testing
+
+inputs:
+  - name: version
+  - name: packaged-product
+  - name: tvs-cli
+run:
+  path: bash
+  args:
+  - -c
+  - |
+    set -eux
+
+    VERSION="$(cat version/version)"
+    RELEASE_LINE="$(echo $VERSION | rev | cut -d'.' -f2- | rev)"
+
+    mkdir -p platform-automation-product
+    tar -xf packaged-product/artifacts*.tar.gz -C platform-automation-product
+
+    mkdir -p platform-automation-image
+    tar -xf platform-automation-product/platform-automation-image-$VERSION.tgz -C platform-automation-image
+    syft platform-automation-image -o cyclonedx-json=platform-automation-image-sbom
```

Comment:
What's going on with the `=platform-automation-image-sbom` syntax? I'm assuming that it's putting the cylonedx-json into a file named `platform-automation-image-sbom`. But then I dond't see that file being used later on.

Reply:
Same thing with platform-automation-vsphere-image-sbom.

Reply:
the `cyclonedx-json` indicates the format of the output and the `platform-automation-image-sbom` is the file to write the output to

Reply:
> the `cyclonedx-json` indicates the format of the output and the `platform-automation-image-sbom` is the file to write the output to

I'm wondering when that output is used. And I don't understand how the manifest.yml works. In the manifest, what does the `bom-for-scanner` refer to? Is that the sbom file? If yes, then I would've expected it to be `platform-automation-image-sbom` (the output of syft...), instead of `./platform-automation-image/sbom`

Reply:
Thank you for catching that, it was one of the refactors I made and managed to miss the rename. I fixed the reference to it in the `manifest` file that is generated below 🙏

Reply:
Solid!